### ISSUE TITLE
sticky group header

### ISSUE THREAD

**Issue by YouSour** [Daniel-Ioannou/flutter_group_list_view#12]
@Daniel-Ioannou hi , Do have any plan add this feature that allow us set sticky group header ? thank in advance 😄

**Comment by Daniel-Ioannou** [comment]
Hi, it's in my plans, but I'm busy right now. As soon as I have more time I will implement this.

**Comment by YouSour** [comment]
ok , nice package, i'm waiting for it @Daniel-Ioannou

**Comment by Shamsudeen12** [comment]
@Daniel-Ioannou can I work on this?

**Comment by Daniel-Ioannou** [comment]
@Shamsudeen12 Yes, Contributions of any kind are more than welcome! Feel free to implement this feature and make a pull request.

**Comment by Shamsudeen12** [comment]
Awesome @Daniel-Ioannou

**Comment by CoderWeiLee** [comment]
Is this function implemented now?
